test/test_formats.py

```diff
@@ -126,265 +126,267 @@ def save_and_load_metadata(filename, metadata):
     'website': 'http://example.com',
 }
 
+# prevent unittest to run tests in those classes
+class CommonTests:
```

Comment:
We have two ways to approaching this without a wrapper class - 
1. Make a mixin
2. Set `__test__` to False

Reply:
Mixin can work, but using a wrapper class is even simpler, imho.

`__test__` approach isn't really documented, i'd stay away from it.

Reply:
I tested the `__test__ = False` approach, it doesn't work.